### PR TITLE
Release v3.2.0

### PR DESCRIPTION
**Pull request by matheusneder**

### CODE REVIEW

File: Source/Otc.AspNetCore.ApiBoot/ApiBootStartup.cs [hunk 1]
```diff
@@ -43,16 +44,24 @@ protected ApiBootStartup(IConfiguration configuration)
...
 
-        private string BuildVersion { get; set; } = string.Empty;
+        private string BuildId
+        {
+            get
+            {
+                if (File.Exists(buildIdFilePath))
+                {
+                    return File.ReadAllText(buildIdFilePath);
+                }
```
Comment (allanpedroni) [inline Source/Otc.AspNetCore.ApiBoot/ApiBootStartup.cs]: I was wondering if these lines can fit some concurrency using file situation. Is it possible at this point or the `lock` solution can treat this case?

File: Source/Otc.AspNetCore.ApiBoot.TestHost/Otc.AspNetCore.ApiBoot.TestHost.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <Authors>Ole Consignado</Authors>
     <Copyright>Ole Consignado (c) 2018</Copyright>
-    <VersionPrefix>3.1.4</VersionPrefix>
+    <VersionPrefix>3.2.0</VersionPrefix>
```
Comment (allanpedroni) [inline Source/Otc.AspNetCore.ApiBoot.TestHost/Otc.AspNetCore.ApiBoot.TestHost.csproj]: **assembly version**
Is there any solution that we can increase automatically an assembly version without manual retouching?

File: Source/Otc.AspNetCore.ApiBoot/Otc.AspNetCore.ApiBoot.csproj [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <Authors>Ole Consignado</Authors>
     <Copyright>Ole Consignado (c) 2018</Copyright>
-    <VersionPrefix>3.1.4</VersionPrefix>
+    <VersionPrefix>3.2.0</VersionPrefix>
```
Comment (allanpedroni) [inline Source/Otc.AspNetCore.ApiBoot/Otc.AspNetCore.ApiBoot.csproj]: see comments in **assembly version**

File: Source/Otc.AspNetCore.ApiBoot/ApiBootStartup.cs (hunk 1 above)
Comment (matheusneder) [reply Source/Otc.AspNetCore.ApiBoot/ApiBootStartup.cs]: There's no concurrency while startingup!

File: Source/Otc.AspNetCore.ApiBoot.TestHost/Otc.AspNetCore.ApiBoot.TestHost.csproj (hunk 2 above)
Comment (matheusneder) [reply Source/Otc.AspNetCore.ApiBoot.TestHost/Otc.AspNetCore.ApiBoot.TestHost.csproj]: Sure, would be great, lets look forward.

**Pull request merged**